packages/clerk-js/src/ui/components/UserProfile/UserProfileAvatarUploader.tsx

```diff
@@ -11,6 +11,7 @@ export const UserProfileAvatarUploader = (
     <AvatarUploader
       {...rest}
       title={localizationKeys('userProfile.profilePage.imageFormTitle')}
+      hasImageUrl={!new RegExp('gravatar').test(user.profileImageUrl || '')}
```

Comment:
Would this be enough?
```suggestion
      hasImageUrl={(user.profileImageUrl || '').includes('gravatar')}
```

Reply:
@nikosdouvlis yes this will do 🙃